libethereum/Account.cpp

```diff
@@ -81,6 +81,50 @@ PrecompiledContract createPrecompiledContract(js::mObject& _precompiled)
 }
 
 }
+namespace
+{
+	string const c_wei = "wei";
+	string const c_finney = "finney";
+	string const c_balance = "balance";
+	string const c_nonce = "nonce";
+	string const c_code = "code";
+	string const c_storage = "storage";
+	string const c_shouldnotexist = "shouldnotexist";
+	string const c_precompiled = "precompiled";
+	std::set<string> const c_knownAccountFields = {
+		c_wei, c_finney, c_balance, c_nonce, c_code, c_storage, c_shouldnotexist,
+		c_code, c_precompiled
+	};
+	void validateAccount(js::mObject const& o)
+	{
+		for (auto const& field: o)
+			if (c_knownAccountFields.find(field.first) == c_knownAccountFields.end())
+			{
+				string const comment = "Unrecognized account field: " + field.first;
+				cerr << comment << "\n";
+				BOOST_THROW_EXCEPTION(UnknownField() << errinfo_comment(comment));
+			}
+	}
+	void validateAccounts(js::mObject const& o)
+	{
+		for (auto const& account: o)
+			validateAccount(account.second.get_obj());
+	}
+	string const c_alloc = "alloc";
+	string const c_accounts = "accounts";
+	void validateAccountMapObj(js::mObject const& o)
+	{
+		for (auto const& field: o)
+		{
+			if (field.first == c_alloc)
```

Comment:
This I guess can be simplified to `if (field.first == c_alloc || field.first == c_accounts)`

Reply:
Also, does anyone still uses these `alloc` and `account` options?

Reply:
I tried removing these.